extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/SerializableTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.kinesis;
+
+import com.hazelcast.test.archunit.ArchUnitRules;
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import org.junit.Test;
+
+import static com.tngtech.archunit.core.importer.ImportOption.Predefined.DO_NOT_INCLUDE_JARS;
+import static com.tngtech.archunit.core.importer.ImportOption.Predefined.DO_NOT_INCLUDE_TESTS;
+
+public class SerializableTest {
```

Comment:
BTW should I mark the test with these annotations?
```
@RunWith(HazelcastParallelClassRunner.class)
@Category({QuickTest.class, ParallelJVMTest.class})
```

What's the heuristic of applying them?

Reply:
Use `HazelcastParallelClassRunner` for running methods in parallel inside single jvm - such tests must not use real network with conflicting ports and must not use any static resources, basically perfectly isolated

use `@Category(ParallelJVMTest.class)` to run test classes in parallel with other test classes (in different jvm) - these must not use real network, but can use static resources because they are inside a separate jvm

`QuickTest` was used to mark test for the PR builder, now when there is no annotation it is automatically. There is a pending PR to remove it, but there was some issue with test hierarchies I think.